bitcoin-magazine/bitcoin-2024/making-bitcoin-more-private-with-cisa.md

```diff
@@ -1,71 +1,76 @@
 ---
-title: "Making Bitcoin More Private with CISA"
-transcript_by: kouloumos via tstbtc v1.0.0 --needs-review
-media: https://www.youtube.com/watch?v=HvI7NPI_Pk0
-tags: ['cisa']
-speakers: ['Fabian Jahr', 'Jameson Lopp', 'Craig Raw']
-date: 2024-08-29
-summary: "Delve into the world of Cross-Input Signature Aggregation (CISA) with our expert panel from Sparrow Wallet, Bitcoin Core, and Casa. This discussion illuminates how CISA could revolutionize Bitcoin transactions by enhancing privacy and reducing fees. Uncover the technical aspects of half and full signature aggregation and understand their potential impact on the Bitcoin network."
+title: 'Making Bitcoin More Private with CISA'
+transcript_by: 'markon1-a via review.btctranscripts.com'
+media: 'https://www.youtube.com/watch?v=HvI7NPI_Pk0'
+date: '2024-08-29'
+tags:
+  - 'cisa'
+speakers:
+  - 'Fabian Jahr'
+  - 'Jameson Lopp'
+  - 'Craig Raw'
+summary: 'Delve into the world of Cross-Input Signature Aggregation (CISA) with our expert panel from Sparrow Wallet, Bitcoin Core, and Casa. This discussion illuminates how CISA could revolutionize Bitcoin transactions by enhancing privacy and reducing fees. Uncover the technical aspects of half and full signature aggregation and understand their potential impact on the Bitcoin network.'
 ---
-Speaker 0: 00:00:01
+Fabian Jahr: 00:00:01
 
 So there's one person on the stage here that is not up there.
 Why?
 
-Speaker 1: 00:00:07
+Nifi: 00:00:07
```

Comment:
This is "Lisa Neigut" aka Nifty, so sometimes you will hear calling her Nifty

Reply:
This I couldn't figure out.